packages/expo-in-app-purchases/ios/EXInAppPurchases/EXInAppPurchasesModule.m

```diff
@@ -0,0 +1,346 @@
+// Copyright 2018-present 650 Industries. All rights reserved.
+
+#import <EXInAppPurchases/EXInAppPurchasesModule.h>
+
+#define QUERY_HISTORY_KEY @"QUERY_HISTORY"
```

Comment:
So far I think the convention is to use `static <type> const EX…` for such constants

https://github.com/expo/expo/blob/8d441c7e9b081cfdd6a45edc165251d7273225a0/packages/expo-av/ios/EXAV/Video/EXVideoView.m#L12